src/qforte/ucc/uccnpqe.py

```diff
@@ -135,7 +135,7 @@ def print_options_banner(self):
             print('Measurement varience thresh:             ',  0.01)
 
         res_thrsh_str = '{:.2e}'.format(self._res_vec_thresh)
-        print('DIIS maxiter:                            ',  self._diis_maxiter) # RENAME
+        print('DIIS maxiter:                            ',  self._opt_maxiter) # RENAME
```

Comment:
Do we still need the `RENAME` comment at the end of this line or is this referring to the renaming that you already took care of?

Reply:
I have absolutely no idea. I'll remove it.